src/go/configmanager/config_manager_test.go

```diff
@@ -1565,13 +1566,23 @@ func TestServiceConfigAutoUpdate(t *testing.T) {
 	newRolloutID = newConfigID
 	testCase := struct {
 		desc                  string
+		fakeOldScReport       string
+		fakeNewScReport       string
 		fakeOldServiceRollout string
 		fakeNewServiceRollout string
 		fakeOldServiceConfig  string
 		fakeNewServiceConfig  string
 		BackendAddress        string
 	}{
 		desc: "Success for service config auto update",
+		fakeOldScReport: fmt.Sprintf(`{
+                "serviceConfigId": "%s",
+                "serviceRolloutId": "%s"
+            }`, oldRolloutID, oldConfigID),
+		fakeNewScReport: fmt.Sprintf(`{
+                "serviceConfigId": "%s",
+                "serviceRolloutId": "%s"
+            }`, newRolloutID, newConfigID),
 		fakeOldServiceRollout: fmt.Sprintf(`{
```

Comment:
fakeOldSrcReport and fakeNewSrReport are generated, do they have to be in the testCase struct?  Can they be just a variable?

Reply:
Putting in the struct is consistent with other serviceRollout/serviceConfig layouts, which make testcases clear.

Reply:
OK, I am fine either way.